DOCUMENTATION.md

```diff
@@ -29,6 +30,12 @@ This module implements all models for Resin Python SDK.
 ## Application
 
 This class implements application model for Resin Python SDK.
+
+Due to API changes, the returned Application object schema has changed. Here are the formats of the old and new returned objects.
```

Comment:
Hmmm, seems like these property-lists are only _partially_ alphabetical? (which IMHO is a bit confusing?)
I wrote a mini Python function to check them ;-)
```python
def check_list(old_list):
    new_list = ', '.join(sorted(old_list.split(', ')))
    if new_list != old_list:
        print("List should be:\n%s" % new_list)
```

Reply:
@lurch, it's weird since basically I did the same, the only difference is I used `list.sort()` and print the sorted list instead. From what I understand, `list.sort()` mutates the list while `sorted` returns a new list.